c2corg_ui/templates/waypoint/edit.html

```diff
@@ -7,139 +9,774 @@
+                <p ng-message="max"  style="{{ (editForm.longitude.$invalid && editForm.longitude.$touched) ? 'display:inline' : ''}}" translate>This field must be between -180° and 180°.</p>
+              </div>
+            </div>
+
+            ## LATITUDE
+            <div id="latitude-group" class="form-group data half"
+                 ng-class="{ 'has-error': editForm.latitude.$touched && editForm.latitude.$invalid,
+                                      'has-success': editForm.latitude.$valid }">
+              <label><span translate>Latitude</span> *</label>
+              <div class="input-group">
+                <input type="number" name="latitude" ng-model="waypoint.lonlat.latitude" ng-change="editCtrl.updateMap()" class="form-control" required required min="-90" max="90" />
+                <span class="input-group-addon">°N</span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.latitude.$valid"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.latitude.$invalid && editForm.latitude.$touched"></span>
+              <div class="help-block" ng-messages="editForm.latitude.$error">
+                <p ng-message="required"  style="{{ (editForm.latitude.$invalid && editForm.latitude.$touched) ? 'display:inline' : ''}}" translate>This field is required.</p>
+                <p ng-message="number"  style="{{ (editForm.latitude.$invalid && editForm.latitude.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+                <p ng-message="min"  style="{{ (editForm.latitude.$invalid && editForm.latitude.$touched) ? 'display:inline' : ''}}" translate>This field must be between -90° and 90°.</p>
+                <p ng-message="max"  style="{{ (editForm.latitude.$invalid && editForm.latitude.$touched) ? 'display:inline' : ''}}" translate>This field must be between -90° and 90°.</p>
+              </div>
+            </div>
+
+            <app-map app-map-edit-ctrl="editCtrl" app-map-draw-type="Point" class="map-edit"></app-map>
+
+            ## MAPS REF
+            <div  id="maps_info-group"  class="input-group">
+              <label translate>maps_references</label>
+              <div ng-repeat="map in waypoint.maps" class="map-info">
+                <input  value="{{map.code}} - {{map.editor}} -  {{map.locales[0].title}}" class="form-control">
+                <span class="glyphicon glyphicon-trash"></span>
+              </div>
+              <div>
+                <input type="text" name="maps_info" ng-model="waypoint.maps_info" class="form-control" placeholder="{{maps_info}}"/>
+                <button id="document-add-maps" type="button" class="btn btn-default" ng-click="editCtrl.addAnotherMapsInfo()"><span class="glyphicon glyphicon-plus"></span></button>
```

Comment:
Why do you need this? "Maps info" is a simple text attribute. I don't think it's worth complicating the HTML code and editing controller by adding line for every maps.